pick_kernel/subkernels.py

```diff
@@ -10,6 +12,13 @@ def __init__(self):
     def register(self, name, subkernel):
         self._subkernels[name] = subkernel
 
+    def register_entry_points(self):
```

Comment:
You may want a unittest for this one -- I found that helped in papermill

Reply:
Great, I'll do that.